include/oneapi/dpl/internal/common_config.h

```diff
@@ -16,7 +16,7 @@
 #ifndef _ONEDPL_COMMON_CONFIG_H
 #define _ONEDPL_COMMON_CONFIG_H
 // Workarounds for libstdc++9, libstdc++10 when new TBB version is found in the environment
-#if __cplusplus >= 201703L
+#if __cplusplus >= 201703L                                                  // KSATODO: leve as is: https://www.intel.com/content/www/us/en/developer/articles/release-notes/intel-oneapi-threading-building-blocks-release-notes.html
```

Comment:
Same again, the check guards `__has_include` and is supposed to always succeed with C++17.

Reply:
Fixed.